internal/engine/buildcontroller.go

```diff
@@ -147,9 +152,8 @@ func (c *BuildController) logBuildEntry(ctx context.Context, entry buildEntry) {
 	}
 }
 
-func getNewManifestFromTiltfile(ctx context.Context, name model.ManifestName) (model.Manifest, model.YAMLManifest, error) {
-	// Sends any output to the CurrentBuildLog
-	t, err := tiltfile.Load(ctx, tiltfile.FileName)
+func getNewManifestFromTiltfile(ctx context.Context, allManifestNames []string, name model.ManifestName) (model.Manifest, model.YAMLManifest, error) {
```

Comment:
it seems wrong for the first arg to be strings and the second arg be a ManifestName. shouldn't they both be ManifestNames?